<ui version="4.0" >
 <class>CoreAccountEditDlg</class>
 <widget class="QDialog" name="CoreAccountEditDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Core Account</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="0" >
      <widget class="QLabel" name="portLabel" >
       <property name="text" >
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="hostLabel" >
       <property name="text" >
        <string>Host:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="passwdLabel" >
       <property name="text" >
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="hostEdit" />
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="accountEdit" />
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="userEdit" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="accountLabel" >
       <property name="text" >
        <string>Account:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="passwdEdit" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QSpinBox" name="port" >
       <property name="minimum" >
        <number>1024</number>
       </property>
       <property name="maximum" >
        <number>65535</number>
       </property>
       <property name="value" >
        <number>4242</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="userLabel" >
       <property name="text" >
        <string>User:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>accountEdit</tabstop>
  <tabstop>hostEdit</tabstop>
  <tabstop>port</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>passwdEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
